Communication/Packets/Incoming/Catalog/CheckGnomeNameEvent.cs

```diff
@@ -48,11 +49,9 @@ public void Parse(GameClient session, ClientPacket packet)
         var y = item.GetY;
 
         //Quickly delete it from the database.
-        using (var dbClient = _database.GetQueryReactor())
+        using (var connection = _database.Connection())
         {
-            dbClient.SetQuery("DELETE FROM `items` WHERE `id` = @ItemId LIMIT 1");
-            dbClient.AddParameter("ItemId", item.Id);
-            dbClient.RunQuery();
+            connection.Execute("DELETE FROM `items` WHERE `id` = @ItemId LIMIT 1", new { ItemId = item.Id});

```

Comment:
Please user lowerPascalCase for variable names including query parameters. This case should be itemId or id would suffice too.